domains/letum.json

```diff
@@ -0,0 +1,10 @@
+{
+   "owner": {
+      "username": "amor_amicae",
+      "discord": "1252631745514180790",
+      "email": ""
+  },
+  "record": {
+    "CNAME": "eqqnfxbx2gfatttkjznoyfryfsryp8llcuicn2swuprsnrho1kihcu9kk3zbcrc.cloudflare-539.workers.dev"    
```

Comment:
Why does it redirect to google?
Sorry but I want a preview of your website.

Reply:
oh its a api
message me on discord